pubtools/_pulp/tasks/clear_repo.py

```diff
@@ -73,6 +73,8 @@ def add_args(self):
             "--content-type",
             help="remove only content of these comma-separated type(s)",
             type=str,
```

Comment:
Didn't notice this until after submit - I wonder does the 'type' argument do anything now?

If it has any effect now, it's probably meant to be `list`.  But maybe it just does nothing once you start using `action`.

Reply:
It looks like using `type=str` isnt needed since values are [read in as strings by default](https://docs.python.org/3/library/argparse.html#type), but I don't mind it. It doesn't hurt to be explicit about the type of the parsed values.

Reply:
Well, the point is that parsed.content_type used to return a str, so that was consistent with type, but now it returns a list, so that's inconsistent with type.

Is it correct that type now means "type of elements within list"?   Docs don't say anything about how `type` and `action` work together as far as I can tell.

If you find for example that setting type=int doesn't cause any tests to fail, I'd take that as evidence that the argument has no effect now and in that case I'd be in favor of removing it, because I think it just misleads the reader into thinking that it has some effect.

Reply:
I think I see what you mean. `ClearRepo.content_type` returns a `list` or `None` as it did before, but `ClearRepo.args.content_type` now returns a `list` instead of a comma-separated `str`.

I did not check for access to `ClearRepo.args.content_type` directly, but no tests failed with that change so we might be safe.

I'd be a fan of adding an additional property to the `ClearRepo` API to return the comma-separated string if it's a common need. But I can also understand the case to remove the `type` argument all together for the reason you mentioned.